### DISCUSSION TITLE
what it the name of target "folder group app pinned" ? for change pls ?

### DISCUSSION THREAD

**Discussion by acharknews** [ramensoftware/windhawk#331] [Q&A]
Hi

im not programmer

i use :

openshell +mika +theme niivu anddromeda dark + windhawk

in windhawk i try to change the "pînned app folder" or ""groupe app"  to make it apear dierctly near the start menu !

i nead to know the name of target of the group apps / or app folders by caterogy ! 

wht it the name of the "target" ? for to change the code or hook in the mod  "Windows 11 Start Menu Styler"

i try to make it "deployed permanent" in the right of start menu separated by empty area and launching app in 2 clics only !

![pinned-app-folder-quetion-01](https://github.com/user-attachments/assets/c1e35278-2e66-4e7d-9087-1b041a24af01)

i want to edit the mod inwindhawk for to make the "folder group app" aprear in th part righ tside of startmenu ! i dont know the name of target like this images bellow, thank you" 

![pinned-app-folder-quetion-02](https://github.com/user-attachments/assets/0d704aaf-5743-4e5e-8aaa-c2b2e3cd7922)

**Comment by m417z** [comment]
Converted to an issue in [windows-11-start-menu-styling-guide](https://github.com/ramensoftware/windows-11-start-menu-styling-guide), which is a better place for this question.
